src/julia/Integer.jl

```diff
@@ -402,7 +402,7 @@ function ispower_moduli(a::Integer, n::Int)
    end
 
    for (p, q_residues) in _p_q_residues
-     if (n % p) == 0
+     if iszero(n % p)
```

Comment:
```suggestion
     if is_divisible_by(n, p)
```
I think this should be even better, right?